website/blog/2022-11-22-move-spreadsheets-to-your-dwh.md

```diff
@@ -0,0 +1,191 @@
+
+### Summary table
+
+| Option/connector | Good for frequently updated data | Good for change tracking | Configurable data types |
+| --- | --- | --- | --- |
+| dbt seeds | ❌ | ✅ | ✅ |
+| Fivetran Browser Upload | ✅ | ❌ | ✅ |
+| Fivetran Google Sheets connector | ✅ | ❌ | ❌ |
+| Fivetran Google Drive connector | ❌ | ❌ | ❌ |
+| Stitch Google Sheets integration | ✅ | ❌ | ❌ |
+| Airbyte Google Sheets connector | ✅ | ❌ | ❌ |
+
+### Fivetran browser upload
+
+[Fivetran’s browser uploader](https://fivetran.com/docs/files/browser-upload) does exactly what it says on the tin: you upload a file to their web portal and it creates a table containing that data in a predefined schema in your warehouse. With a visual interface to modify data types, it’s easy for anyone to use. And with an account type with the permission to only upload files, you don’t need to worry about your stakeholders accidentally breaking anything either.
+
+<Lightbox src="/img/blog/2022-11-22-move-spreadsheets-to-your-dwh/fivetran-uploader-1.png" title="Converting data types from text to dates and numbers is easy in the visual editor" />
+
+<Lightbox src="/img/blog/2022-11-22-move-spreadsheets-to-your-dwh/fivetran-uploader-2.png" title="Picking the matching date format from a list of options to convert them to a standardized format" />
+
+A nice benefit of the uploader is support for updating data in the table over time. If a file with the same name and same columns is uploaded, any new records will be added, and existing records (per the <Term id="primary-key"/>) will be updated.
+
+However, keep in mind that there is no source control on these changes or a to revert them; you might want to consider [snapshotting changes](https://docs.getdbt.com/docs/building-a-dbt-project/snapshots) in dbt if that’s a concern.
+
+Also, Fivetran won’t delete records once they’re created, so the only way to remove records created using this process is by manually [deleting](https://docs.getdbt.com/terms/dml#delete) them from your warehouse. If you have an ad-hoc connector, consider having an automated process to drop these tables regularly, especially if you have PII management concerns.
+
+#### Good fit for:
+
+- Files that are frequently updated by someone
+- Allowing anyone in the company to upload files
+- Ad-hoc data loads
+- Updating a table instead of creating a new one
+- Basic data type changes (including handling currency columns)
+- Larger files
+
+#### Not a good fit for:
+
+- Tracking changes to data
+- Complex type mappings
+
+### Fivetran Google Sheets connector
+
+The main benefit of connecting to Google Sheets instead of a static spreadsheet should be obvious—teammates all over the world can change the sheet and new records will be loaded into your warehouse automatically. [Fivetran’s Google Sheets connector](https://fivetran.com/docs/files/google-sheets) requires some additional implementation details, but collaborative editing can make the additional configuration effort worthwhile.
```

Comment:
```suggestion
The main benefit of connecting to Google Sheets instead of a static spreadsheet should be obvious—teammates can change the sheet from anywhere and new records will be loaded into your warehouse automatically. [Fivetran’s Google Sheets connector](https://fivetran.com/docs/files/google-sheets) requires some additional initial configuration, but collaborative editing can make the effort worthwhile.
```